node/cn/api_backend.go

```diff
@@ -278,8 +278,14 @@ func (b *CNAPIBackend) ProtocolVersion() int {
 	return b.cn.ProtocolVersion()
 }
 
+// If current block is magma hard forked, it returns the baseFee * 2.
+// Other cases, it returns the unitPrice.
 func (b *CNAPIBackend) SuggestPrice(ctx context.Context) (*big.Int, error) {
-	return b.gpo.SuggestPrice(ctx)
+	suggestedPrice, _ := b.gpo.SuggestPrice(ctx)
+	if b.ChainConfig().IsMagmaForkEnabled(new(big.Int).Add(b.CurrentBlock().Number(), common.Big1)) {
+		return new(big.Int).Mul(suggestedPrice, common.Big2), nil
+	}
+	return suggestedPrice, nil
```

Comment:
This function now always returns error=nil. Is this ok?

Reply:
Yes, actually gpo.SuggestPrice always return nil as well.